corehq/apps/aggregate_ucrs/models.py

```diff
@@ -110,6 +111,16 @@ def get_primary_column_adapters(self):
         for primary_column in self.primary_columns.all():
             yield PrimaryColumnAdapter.from_db_column(primary_column)
 
+    @property
+    def pk_columns(self):
+        columns = []
+        for col in self.get_columns():
+            if col.is_primary_key:
+                column_name = col.database_column_name
+                if isinstance(column_name, bytes):
+                    column_name = column_name.decode('utf-8')
+                columns.append(column_name)
+
```

Comment:
Is this missing a `return columns`?

Reply:
Ha, i think I added that at the exact same time you made this comment

Reply:
lol